docs/02-Legacy/breakdown.md

```diff
@@ -3,6 +3,302 @@ id: breakdown
+| Pin   | 6      | `ads2` | `ADS.P1`   | `True`        | `32000` (Default) |
+| Pin   | 7      | `ads2` | `ADS.P2`   | `True`        | `32000` (Default) |
+| Pin   | 8      | `ads2` | `ADS.P3`   | `True`        | `32000` (Default) |
+| Pin   | 9      | `ads3` | `ADS.P0`   | `True`        | `25000`           |
+
+If `pi_id == 2`, returns
+
+| Class | Pin id | ADC ID | ADC Pin id | record_values | threshold         |
+| ----- | ------ | ------ | ---------- | ------------- | ----------------- |
+| Pin   | 10     | `ads1` | `ADS.P0`   | `True`        | `15000`           |
+| Pin   | 11     | `ads1` | `ADS.P1`   | `True`        | `32000` (Default) |
+| Pin   | 12     | `ads1` | `ADS.P2`   | `True`        | `32000` (Default) |
+| Pin   | 13     | `ads1` | `ADS.P3`   | `True`        | `21000`           |
+| Pin   | 14     | `ads2` | `ADS.P0`   | `True`        | `32000` (Default) |
+| Pin   | 15     | `ads2` | `ADS.P1`   | `True`        | `32000` (Default) |
+| Pin   | 16     | `ads2` | `ADS.P2`   | `True`        | `32000` (Default) |
+| Pin   | 17     | `ads2` | `ADS.P3`   | `True`        | `32000` (Default) |
+| Pin   | 18     | `ads3` | `ADS.P0`   | `True`        | `32000` (Default) |
+
+**Otherwise, raises an error**
+
+:::info
+As their name implies, `ADS.P0`, `ADS.P1`, `ADS.P2` and `ADS.P3` corresponds to the pin values 0, 1, 2, and 3 respectively, and are `int` values `0`, `1`, `2`, and `3`.
+:::
+
+:::note
+`ads1`, `ads2` and `ads3` are ADS ids hardcoded as:
+
+```
+ads1 = ADS.ADS1115(i2c, address=0x48)
+ads2 = ADS.ADS1115(i2c, address=0x49)
+ads3 = ADS.ADS1115(i2c, address=0x4a)
+```
+
+:::
+
+---
+
+### `runner.py` (_TODO_)
+
+> Importantly, `runner.py` takes a `module path` and `function` and runs `function` in a process. Upon the process terminating, `module path` is reloaded and the new `function` is run in a new process. **This is used to run main.py**
+
+### `flagger.py` (_TODO_)
+
+> Previous repo says that it is a small ecosystem to update the code on the pi every time a new comit is made.
+
+---
+
+### `*.log`
+
+`main.py` writes logs with the `logging` module to the directory `~/laundro_logs/<pi ID>/`.
+
+A log file can be read with live updates using:
+
+```bash
+tail -f <name of log file>
+```
+
+| Log File   | Description                                 |
+| ---------- | ------------------------------------------- |
+| `all.log`  | Contains all log messages                   |
+| `info.log` | Contains only INFO level and above messages |
+
+**Practically speaking, log messages for individual sensor readings will be in `all.log` and not `info.log`**
+
+> sample `*.log` files can be find in the logs dump [here](#assets)
+
+### `firestore_manager.py` (_TODO_)
+
+### `pin.py`
+
+**Contains the class definition for `Pin`.**
+
+Each `Pin` object represents a pin on the **raspberry pi** (not the Analog to Digital Conveter (ADC)), and has the following properties and methods:
```

Comment:
not sure if i misunderstood their implementation but looking at the class methods, it does seem like each `Pin` object refers to the input pin to the respective ADS

https://github.com/usdevs/laundry-pi/blob/795ada7e6333fc3f58388bd3b1f9940a62d64225/pin.py#L35-L43

since the class method reads values from the ADS pins

Reply:
im guessing they have a id param in `Pin` so that they can differentiate the different ADC pins since they are using 3 separate ADCs

Reply:
Ayy thanks for this, took a while to think about this and reply, I'd say that there is no difference between ADC pins and Rpi pins, so there ought to be a correction there, I think how it is is that you read the Rpi pins by reading values from ADC pins, wdy think?

Reply:
I would say it's more of you are reading the input pins of the ADC using the I2C bus (with SDA/SCL pins) of the RPI. 

Because we can technically read inputs of multiple ADCs using the same I2C bus (1 SDA and 1 SCL) of the RPI